config/legacyconfig/migrate_test.go

```diff
@@ -89,6 +104,9 @@ plugin:
   statefulSet:
     scheduling:
       podManagementPolicy: OrderedReady
+external:
+  platform:
+    apiKey: null
```

Comment:
That also doesn't look correct to me, why is this now always null? Is that expected?